annlite/container.py

```diff
@@ -30,6 +30,7 @@ def __init__(
         serialize_config: Optional[Dict] = None,
         data_path: Path = Path('./data'),
         lock: bool = True,
+        hnsw_using_pq: Optional['PQCodec'] = None,
```

Comment:
I suggest reusing `pq_codec` parameter? I does not make sense to introduce a new parameter has the same role

Reply:
The problem here is HNSW backend is enabled only when `pq_codec is None`.
https://github.com/jina-ai/annlite/blob/5225e6cbec7df3fbf4dbd7ab404d34bb7d899a29/annlite/container.py#L40-L62

Reply:
That's fine to change the logic if it does not make sense any more.

Reply:
So we remove the use of `PQIndex` here?